### PR TITLE
517 schul anmeldelink in admin area generieren

### PR DESCRIPTION
**Pull request by pinussilvestrus**

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
[Schul-Anmeldelink in Admin-Area generieren](https://trello.com/c/MgTazx7I/517-schul-anmeldelink-in-admin-area-generieren)

**Comment by pinussilvestrus** [comment]
image

File: controllers/link.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const api = require('../api');
+
+router.post('/', function (req, res, next) {
+    let target = `${req.protocol}://${req.get('host')}/register/${req.body.schoolId}`;
```
Comment (MasterCarl) [inline controllers/link.js]: I'm not sure if `req.protocol` and `host` always works e.g. behind a proxy

Comment (pinussilvestrus) [inline controllers/link.js]: @MasterCarl changed it in new commit to `req.headers.origin`, worked on production and staging in https://github.com/schul-cloud/schulcloud-client/blob/master/controllers/pwrecovery.js#L12 😄

**Pull request merged**